main.go

```diff
@@ -10,4 +11,6 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
+
+	defer bot.Connect()
```

Comment:
The `bot.Connect` method already relies on a deferable to run the bot. You dont need another `defer` here.

Reply:
Done!